src/App.tsx

```diff
@@ -17,18 +77,33 @@
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                status={filterStatus}
+                searchTerm={searchTerm}
+                onFilterChange={handleFilterChange}
+                onSearch={handleSearch}
+                onClearSearch={handleClearSearch}
+              />
```

Comment:
Verify that the `TodoFilter` component is correctly implemented and imported. Ensure it handles the props `status`, `searchTerm`, `onFilterChange`, `onSearch`, and `onClearSearch` as expected.